src/graphic/text/bidi.cc

```diff
@@ -725,8 +742,16 @@ UI::FontSet const* find_fontset(const char* word, const UI::FontSets& fontsets)
 	UI::FontSets::Selector selector;
 	if (has_script_character(word, UI::FontSets::Selector::kArabic)) {
 		selector = UI::FontSets::Selector::kArabic;
+	} else if (strcmp(word, "简体中文") == 0 || strcmp(word, "繁體中文") == 0) {
```

Comment:
Please comment that these are the locale names. Not sure if the extra cost is worth the performance hit just for the language selection list.